static/data/hierarchy_top.json

```diff
@@ -355,15 +355,24 @@
   "l": "Household",
   "t": "p",
   "e": "Household",
-  "c": 680,
+  "c": 690,
   "cd": [
    {
     "sv": [
      "Count_Household"
     ],
-    "l": "Household",
+    "l": "Count",
     "t": "v",
-    "e": "Household",
+    "e": "Count",
+    "c": 1
+   },
+   {
+    "sv": [
+     "Median_Income_Household"
+    ],
+    "l": "Median Income",
```

Comment:
Median household income, so it won't be confused with median individual income. is this the string that will be used for the chips as well?

Reply:
Done